docs/config/index.md

```diff
@@ -1662,25 +1662,15 @@ Format options for snapshot testing. These options are passed down to [`pretty-f
 ::: tip
 Beware that `plugins` field on this object will be ignored.
 
-If you need to extend snapshot serializer via pretty-format plugins, please, use [`expect.addSnapshotSerializer`](/api/expect#expect-addsnapshotserializer) API.
+If you need to extend snapshot serializer via pretty-format plugins, please, use [`expect.addSnapshotSerializer`](/api/expect#expect-addsnapshotserializer) API or [snapshotSerializers](#snapshotserializers-1-3-0) option.
 :::
 
-### resolveSnapshotPath<NonProjectOption />
```

Comment:
It looks like you removed `resolveSnapshotPath` documentation but is it intentional? (like is this a duplicate?)

Reply:
Thanks for pointing that out! It was my neglect, already reverted it.